src/parser/lexer.rs

```diff
@@ -296,14 +294,14 @@ pub enum StringToken<'input> {
     #[error]
     Error,
 
-    #[regex("[^\"#\\\\]+")]
+    #[regex("[^\"%\\\\]+")]
     // Has lower matching priority than `HashBrace` according to Logos' rules.
-    #[token("#")]
+    #[token("%")]
     Literal(&'input str),
 
     #[token("\"")]
     DoubleQuote,
-    #[token("#{")]
+    #[token("%{")]
     HashBrace,
```

Comment:
```suggestion
    Interpolation,
```
I guess `HashBrace` doesn't make sense anymore.